docs/sources/visualizations/gauge-panel.md

```diff
@@ -53,3 +53,10 @@ Adjust the sizes of the gauge text.
 
 - **Title -** Enter a numeric value for the gauge title size.
 - **Value -** Enter a numeric value for the gauge value size.
+
+## Standard options
+
+Adjust various options.
```

Comment:
With only one setting listed, "various options" doesn't work well here. If I understand the comments, this is changing a default behavior of gauges, so would "Adjust default gauge behaviors" work instead?

Reply:
Hi @gguillotte-grafana thanks for your review.